notebooks/segmentation/segmentation_inspection.ipynb

```diff
@@ -362,6 +392,403 @@
     "b.widget.children[0].description = \"plot traces\""
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "movie_list = list_movie_files(experiment_selector.value)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class VideoGenerator(object):\n",
```

Comment:
It seems that if `VideoGenerator.__init__()` took as an arg `tmpdir` then the class could be moved out of the notebook and into the `video_utils` code